.github/workflows/model_loading.yml

```
@@ -0,0 +1,31 @@
+name: Model Loading
+
+on:
+  pull_request:
+    paths:
+      - 'mteb/models/**.py'
+
+jobs:
+  extract-and-run:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+        cache: 'pip'
+
+    - name: Install dependencies and run tests
+      run: |
+        make install-for-tests
+        pip install ".[pylate,gritlm,xformers,model2vec]"
+    - name: Extract model names
+      run: |
+        python scripts/extract_model_names.py
+    - name: Run tests
+      run: |
+        python tests/test_models/test_model_loading.py
```

```suggestion
    - name: Install dependencies and run tests
      run: |
        make model-load-test
```

to keep them compatible.

---

I separated them for things to be cached and for better debugging experience. Do you think that it's a fair tradeoff?